index.js

```diff
@@ -27,4 +85,67 @@ backAbout.addEventListener('click', () => {
   document.getElementById('main').style.filter = 'none';
-});
\ No newline at end of file
+});
+
+//  work popout
+
+openWork1.addEventListener('click', () => {
+  document.getElementById('display_work').style.display = 'block';
+  document.getElementById('main').style.filter = 'blur(5px)';
+  workName.innerHTML = `${projects[0].name}`;
+  workDescription.innerHTML = `${projects[0].description}`;
+  featuredImage.setAttribute('src', projects[0].image);
+  tech1.innerHTML = `${projects[0].technologies[0]}`;
+  tech2.innerHTML = `${projects[0].technologies[1]}`;
+  tech3.innerHTML = `${projects[0].technologies[2]}`;
+  goLive.setAttribute('href', projects[0].liveLink);
+  goSource.setAttribute('href', projects[0].sourceLink);
+  year.innerHTML = `${projects[0].year}`;
+});
+
+openWork2.addEventListener('click', () => {
+  document.getElementById('display_work').style.display = 'block';
+  document.getElementById('main').style.filter = 'blur(5px)';
+  workName.innerHTML = `${projects[1].name}`;
+  workDescription.innerHTML = `${projects[1].description}`;
+  featuredImage.setAttribute('src', projects[1].image);
+  tech1.innerHTML = `${projects[1].technologies[3]}`;
+  tech2.innerHTML = `${projects[1].technologies[2]}`;
+  tech3.innerHTML = `${projects[1].technologies[1]}`;
+  goLive.setAttribute('href', projects[1].liveLink);
+  goSource.setAttribute('href', projects[1].sourceLink);
+  year.innerHTML = `${projects[1].year}`;
+});
+
+openWork3.addEventListener('click', () => {
+  document.getElementById('display_work').style.display = 'block';
+  document.getElementById('main').style.filter = 'blur(5px)';
+  workName.innerHTML = `${projects[2].name}`;
+  workDescription.innerHTML = `${projects[2].description}`;
+  featuredImage.setAttribute('src', projects[2].image);
+  tech1.innerHTML = `${projects[2].technologies[2]}`;
+  tech2.innerHTML = `${projects[2].technologies[0]}`;
+  tech3.innerHTML = `${projects[2].technologies[1]}`;
+  goLive.setAttribute('href', projects[2].liveLink);
+  goSource.setAttribute('href', projects[2].sourceLink);
+  year.innerHTML = `${projects[2].year}`;
+});
+
+openWork4.addEventListener('click', () => {
+  document.getElementById('display_work').style.display = 'block';
+  document.getElementById('main').style.filter = 'blur(5px)';
+  workName.innerHTML = `${projects[3].name}`;
+  workDescription.innerHTML = `${projects[3].description}`;
+  featuredImage.setAttribute('src', projects[3].image);
+  tech1.innerHTML = `${projects[3].technologies[1]}`;
+  tech2.innerHTML = `${projects[3].technologies[2]}`;
+  tech3.innerHTML = `${projects[3].technologies[3]}`;
+  goLive.setAttribute('href', projects[3].liveLink);
+  goSource.setAttribute('href', projects[3].sourceLink);
+  year.innerHTML = `${projects[3].year}`;
+});
+
+closeWorks.addEventListener('click', () => {
+  document.getElementById('display_work').style.display = 'none';
+  document.getElementById('main').style.filter = 'none';
+});
```

Comment:
- [ ] No doubt, you have made great efforts here, and it's really commendable :1st_place_medal: but there are lots of repetition in the code - `openWork1`-`openWork4` for each card. This does not follow best practices principle of  DRY (Don't Repeat Yourself). To show your expertise as a prolific developer with in-depth understanding of your craft, I think it might be nice to use a `for-loop`, `Array.map()` or a `Array.forEach()` among other means of iterables to traverse your array of objects and apply a template/format to each of them.
- This will save a lot of time and efforts. It'll also reduce any redundant loading probability to increase the performance of your page. :hammer_and_wrench: 
- [ ] Another striking move that I think can make you page more user friendly as well as being impressive to recruiters is to try to inspect the element causing an overflow when in the mobile view of your `pop-up` window. Kindly fix the horizontal scrollbar issue as shown in the screenshot below:

![image](https://user-images.githubusercontent.com/99715640/223840545-6799245f-a14d-4f60-8c0c-525941f7612b.png)